doc/source/user_guide/link_code.rst

```diff
@@ -0,0 +1,47 @@
+.. _ref_user_guide_link_code:
+
+Linkcode extension
+==================
+
+The Linkcode extension automatically adds "View Source" links to the documentation for Python, C, C++, 
+and JavaScript objects. It allows you to link to the source code hosted on GitHub.
+
+Installation
+------------
+
+To use the ansys-sphinx-theme linkcode extension, you need to add it to the Sphinx configuration:
+
+1. Add the following line in the ``conf.py`` file:
```

Comment:
```suggestion
#. Add the following line in the ``conf.py`` file:
```

Automated numbering is preferred... hardcoded number sections end up being unmaintainable.